source/agora/network/Manager.d

```diff
@@ -1243,7 +1249,60 @@ public class NetworkManager
         settings.httpClientSettings.readTimeout = timeout;
         settings.httpClientSettings.proxyURL = this.proxy_url;
 
-        return new RestInterfaceClient!NameRegistryAPI(settings);
+        auto remote =  new RestInterfaceClient!NameRegistryAPI(settings);
```

Comment:
What is we dont have a `remote` but only a local one? L1244 would return early.

Reply:
This would be a welcome change after mentioned issue gets resolved. Even it won't be problem after #2703